.github/workflows/update-flake.yml

```
@@ -57,30 +57,22 @@ jobs:
 
       - name: update lock files
         run: |
-          nix flake update \
-            --commit-lock-file \
-            --option commit-lock-file-summary "flake: update public inputs"
+          nix flake update &
+          nix flake update --flake ./flake/dev &
 
-          nix flake update \
-            --commit-lock-file \
-            --flake ./flake/dev \
-            --option commit-lock-file-summary "flake: update dev inputs"
+          wait
 
           # The nixpkgs maintainers may have changed, so keep all-maintainers
           # in sync
           if nix run .#all-maintainers; then
             git add generated/all-maintainers.nix
-
-            if
-              ! git commit --message "stylix: update all-maintainers list"
-            then
-              echo "::debug::generated/all-maintainers.nix has no changes"
-            fi
-
           else
             echo "::error::failed to update generated/all-maintainers.nix"
           fi
 
+          git add --all
```

I'd prefer to be explicit 
```suggestion
          git add ./flake.lock ./flake/dev/flake.lock
```

---

> I'd prefer to be explicit
> ```suggestion
>           git add ./flake.lock ./flake/dev/flake.lock
> ```

I moved the `git add` to the relevant code:

```diff
diff --git a/.github/workflows/update-flake.yml b/.github/workflows/update-flake.yml
index 6da42d7a..aebf29a6 100644
--- a/.github/workflows/update-flake.yml
+++ b/.github/workflows/update-flake.yml
@@ -62,6 +62,8 @@ jobs:

           wait

+          git add {,flake/dev/}flake.lock
+
           # The nixpkgs maintainers may have changed, so keep all-maintainers
           # in sync
           if nix run .#all-maintainers; then
@@ -70,7 +72,6 @@ jobs:
             echo "::error::failed to update generated/all-maintainers.nix"
           fi

-          git add --all
           git commit --message "flake: update all inputs"

       - name: create pull request
```